source/lib/courses/index.js

```diff
@@ -1,7 +1,3 @@
 // @flow
 export {loadAllCourses} from './load-all-courses'
-export type {
-  CourseType,
-  CourseCollectionType,
-  CoursesByTermType,
-} from './types'
+export type {CourseType, CourseCollectionType, CoursesByTermType} from './types'
```

Comment:
The only explanation I have for these is that … the algorithm was counting the semicolon in the line length? which makes sense.